proxies/live/apiproxy/policies/KeyValueMapOperations.GetSecureVariables.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<KeyValueMapOperations async="false" continueOnError="false" enabled="true" name="KeyValueMapOperations.GetSecureVariables" mapIdentifier="pds-variables-encrypted">
```

Comment:
Yes, `pds-variables-encrypted` is correct even this is SDS. At this moment API-M doesn't have a generic store for `/_status` apikey so the PDS one has to be used